### DISCUSSION TITLE
apps.plugin not support shared memory?

### DISCUSSION THREAD

**Discussion by yezhengmao1** [netdata/netdata#13444] [Q&A] [answered]
why apps.plugin not support  shared memory? and how to trace the  shared memory?

**Comment by vlvkobal** [comment, marked as answer]
It subtracts shared memory from real memory in `*.mem` charts - "Real Memory (w/o shared)". But you are correct, there are no separate charts for RssFile or RssShmem. Do you think we need to add a chart(s) for the shared memory?

**Reply by yezhengmao1** [reply]
Yes, it`s also important.
I found the apps.plugin only gets VmRSS, RssFile, RssShmem, and mem = VmRss - (RssFile + RssShmem), why not get RssAnon directly.
Can add charts for RssFile, RssShmem, and also RssAnon.

**Reply by vlvkobal** [reply]
Could you create an issue with a feature request and describe your proposal in detail? It would be nice if you can come up with chart titles and descriptions since you are competent in the area.

**Reply by yezhengmao1** [reply]
ok. i will create issue with this.
